server/models/migrations/next/migrate.sql

```diff
@@ -39,3 +39,72 @@ ALTER TABLE service DROP COLUMN profit_center_id;
 
 DROP TABLE IF EXISTS profit_center;
 DROP TABLE IF EXISTS cost_center;
+
+-- rewrite PostToGeneralLedger
+DELIMITER $$
+
+DROP PROCEDURE PostToGeneralLedger$$
```

Comment:
Is this supposed to be in the migration file?  :)

Reply:
Since we have `yarn migrate`, you are right, we don't need this code here